### PR TITLE
Misc minor updates (AR, IN, KY, OH, TN)

### PR DESCRIPTION
**Pull request by mapcat**

### CODE REVIEW

File: hwy_data/OH/usaus/oh.us022.wpt [hunk 1]
```diff
@@ -91,8 +91,8 @@ OH723 http://www.openstreetmap.org/?lat=40.012669&lon=-81.624330
 +x90 http://www.openstreetmap.org/?lat=40.014146&lon=-81.617780
 +x91 http://www.openstreetmap.org/?lat=40.024881&lon=-81.605238
```
Comment (rickmastfan67) [inline hwy_data/OH/usaus/oh.us022.wpt:92]: This line needs to be either deleted, or have it's similar line restored in US-40's file, as when it's counterpart in US-40's file was deleted, it broke the multiplex between the two in this area.

File: hwy_data/OH/usaus/oh.us040.wpt [hunk 2]
```diff
@@ -23,80 +24,79 @@ OH369 http://www.openstreetmap.org/?lat=39.921477&lon=-83.913376
...
 +x41 http://www.openstreetmap.org/?lat=39.941244&lon=-81.986992
 OH93_N http://www.openstreetmap.org/?lat=39.948322&lon=-81.960711
-AirRd +OH797 http://www.openstreetmap.org/?lat=39.965584&lon=-81.897916
+AirRd http://www.openstreetmap.org/?lat=39.965584&lon=-81.897916
 I-70(164) http://www.openstreetmap.org/?lat=39.974129&lon=-81.833252
 OH83 http://www.openstreetmap.org/?lat=39.993126&lon=-81.741060
 +x50 http://www.openstreetmap.org/?lat=39.994817&lon=-81.710236
 PetCreRd http://www.openstreetmap.org/?lat=40.010284&lon=-81.656769
 OH723 http://www.openstreetmap.org/?lat=40.012669&lon=-81.624330
-+x60 http://www.openstreetmap.org/?lat=40.014146&lon=-81.617780
```
Comment (rickmastfan67) [inline hwy_data/OH/usaus/oh.us040.wpt:83]: This line needs to be either restored, or have it's similar line deleted in US-22's file, as this broke the multiplex between the two in this area.

File: hwy_data/OH/usaus/oh.us022.wpt [hunk 3]
```diff
@@ -91,8 +91,8 @@ OH723 http://www.openstreetmap.org/?lat=40.012669&lon=-81.624330
 +x90 http://www.openstreetmap.org/?lat=40.014146&lon=-81.617780
```
Comment (rickmastfan67) [inline hwy_data/OH/usaus/oh.us022.wpt:91]: This line needs to be either deleted, or have it's similar line restored in US-40's file, as when it's counterpart in US-40's file was deleted, it broke the multiplex between the two in this area.

**Comment by mapcat** [comment]
Thanks. Deleted in US 22. Also found and corrected another broken concurrency on OH60/146 in Zanesville.

**Pull request merged**